src/pages/ProfilesExplorerView/components/SceneExploreDiffFlameGraph/SceneExploreDiffFlameGraph.tsx

```diff
@@ -79,6 +81,51 @@ export class SceneExploreDiffFlameGraph extends SceneObjectBase<SceneExploreDiff
         this.state.presetsPicker.openSelect();
       })
     );
+
+    this._subs.add(
+      this.subscribeToEvent(EventEnableSyncTimeRanges, (event) => {
+        const { enable } = event.payload;
+        const { baselinePanel, comparisonPanel } = this.state;
+
+        comparisonPanel.toggleTimeRangeSync(enable);
```

Comment:
The state is duplicated - it's always true/false in both. Would it make sense to keep only one variable to store it? Or maybe sync it with URL with a single variable?

Reply:
🤔 I'm not sure to follow... Unless we move it to the URL (but I don't think we should - see my main comment), the state has to be duplicated: both panels need to know if their sync button is active or not.